### PR TITLE
NXP-21486: migrate template rendering to web ui 8.10

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: nuxeo-template-rendering-jsf/pom.xml [hunk 1]
```diff
@@ -8,8 +8,8 @@
   </parent>
 
   <groupId>org.nuxeo.template.rendering</groupId>
-  <artifactId>nuxeo-template-rendering-web</artifactId>
-  <name>Template Rendering Web layer</name>
+  <artifactId>nuxeo-template-rendering-jsf</artifactId>
+  <name>Template Rendering JSF layer</name>
```
Comment (Gabez0r) [inline nuxeo-template-rendering-jsf/pom.xml]: Be aware of bundle rename.

**Comment by efge** [comment]
👍 if this is a pure backport

**Comment by guirenard** [comment]
:+1:

**Comment by efge** [comment]
👍

**Pull request merged**